recipe/meta.yaml

```
@@ -1,46 +1,51 @@
 {% set name = "repoze.lru" %}
 {% set version = "0.7" %}
-{% set file_ext = "tar.gz" %}
-{% set hash_type = "sha256" %}
-{% set hash_value = "0429a75e19380e4ed50c0694e26ac8819b4ea7851ee1fc7583c8572db80aff77" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.{{ file_ext }}
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
-  {{ hash_type }}: {{ hash_value }}
+  fn: {{ name }}-{{ version }}.tar.gz
```

```suggestion
  fn: {{ name }}-{{ version }}.tar.gz
```
Not a blocker, but of curiosity, is it best practice to only keep this `fn` for consistency if a recipe already has it or are we adding it to every recipe?

---

I didn't notice that until now. I suppose it could be deleted.

---

I got rid of a lot of Jinja templating already.

---

Ahh okay, I see. It just popped up in one of my current tickets as well, so I figured I ask. Thanks!